shared/src/metabase/shared/util/currency.cljc

```diff
@@ -634,6 +634,13 @@
           :rounding 0,
           :code "PEN",
           :name_plural "Peruvian nuevos soles"}],
+   [:PGK {:symbol "K",
+          :name "Papua New Guinean kina",
```

Comment:
Looks like we normally capitalize the singular name in other entries. Would it be appropriate to capitalize it here as well?

```suggestion
          :name "Papua New Guinean Kina",
```

Reply:
I usually see it capitalised, but Wikipedia showed it lowercase. Let's capitalise, thanks!